mono/mini/mini-llvm-cpp.cpp

```diff
@@ -21,6 +21,11 @@
 // possible
 //
 
+#ifdef _MSC_VER
+#pragma warning(disable:4141)
```

Comment:
Please add a comment as to what the warnings are. Or tell me just to use the documentation every time and gradually memorize them.